packages/unplugin-typia/tests/fixtures/is.ts

```diff
@@ -1,5 +1,5 @@
 import typia from 'typia';
-import type { IMember } from './type.js';
+import type { IMember } from './type.d.ts';
```

Comment:
Do we need this change? the previous statement does  not work?

Reply:
We don't need it, it was changed by IDE's auto imports

Reply:
Ok!